google-login-plugin/src/com/google/gct/login/stats/GctTracking.java

```diff
@@ -23,13 +23,14 @@
   public static final String CATEGORY = "com.google.cloud.tools";
 
   // actions
-  public static final String NEW_ENDPOINT = "new.endpoint";
   public static final String DEPLOY = "appengine.update";
-  public static final String VCS = "vcs";
   public static final String GRADLE_IMPORT = "appengine.gradle.import";
-  public static final String RUN = "appengine.run";
-  public static final String WIZARD = "appengine.wizard";
+  public static final String LOGIN = "user.login";
```

Comment:
@patflynn  I moved this class to the login plugin so that I could add the LOGIN tracking string, since the core plugin depends on the login plugin. I could move the LOGIN tracking string in another file, maybe called LoginTracking and put GctTracking back in the core plugin.  Would this be better?